# Ex058 - Exception handling (4): try~except Exception as e)

try:
    print(param)
except Exception as e:
    print(e)    # "name 'param' is not defined" is printed out.
